site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -4,15 +4,76 @@ linkTitle: "Helm"
 weight: 20
 description: >
   Install Agones on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
-
 ---
 
 ## Prerequisites
 
-- [Helm](https://docs.helm.sh/helm/) package manager 2.10.0+
+- [Helm](https://helm.sh/) package manager 2.10.0+, or 3.2.3+
 - [Supported Kubernetes Cluster]({{< relref "../_index.md#usage-requirements" >}})
 
-## Installing the Chart
+{{% alert title="Note" color="info"%}}
+We are in the process of shifting our development platform to run on [Helm 3](https://helm.sh/blog/helm-3-released/),
+and recommend upgrading your Helm installs,
+as it will soon become the most tested platform.
+{{% /alert %}}
+
+## Helm 3
+
+### Installing the Chart
+
+To install the chart with the release name `my-release` using our stable helm repository:
+
+```bash
+$ helm repo add agones https://agones.dev/chart/stable
+$ helm install my-release --namespace agones-system --create-namespace agones/agones
+```
+
+_We recommend to install Agones in its own namespaces (like `agones-system` as shown above)
+you can use the helm `--namespace` parameter to specify a different namespace._
+
+When running in production, Agones should be scheduled on a dedicated pool of nodes, distinct from where Game Servers are scheduled for better isolation and resiliency. By default Agones prefers to be scheduled on nodes labeled with `agones.dev/agones-system=true` and tolerates node taint `agones.dev/agones-system=true:NoExecute`. If no dedicated nodes are available, Agones will
+run on regular nodes, but that's not recommended for production use. For instructions on setting up a dedicated node
+pool for Agones, see the [Agones installation instructions]({{< relref "../_index.md" >}}) for your preferred environment.
+
+The command deploys Agones on the Kubernetes cluster with the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
```

Comment:
There's a tip in the helm 2 section about installing a development build. Should we explain how to do that with helm3 as well?

Reply:
Actually, I probably should have deleted both. It reads like you can use the standard chart with a dev image -- which may or may not be the case.

I felt it was better to remove the block as it could be a bad path for people to go down (and the instructions on a PR are better). 

Should I delete both, WDYT?

Reply:
If we don't verify that it keeps working, we should delete both (but replace it with something in the developer docs in github so that devs still know how to install pre-release versions via helm).

Reply:
> replace it with something in the developer docs in github so that devs still know how to install pre-release versions via helm

Yeah, we have that with `make install`, also we have the instructions on each Pull Request CI success comment as well.

Reply:
That sounds sufficient.